packages/react-native-editor/__device-tests__/helpers/caps.js

```diff
@@ -11,6 +11,7 @@ const ios = {
 exports.iosLocal = ( { iPadDevice = false } ) => ( {
 	...ios,
 	deviceName: ! iPadDevice ? 'iPhone 13' : 'iPad Pro (9.7-inch)',
+	devicePixelRatio: ! iPadDevice ? 3 : 2,
```

Comment:
This approach looks good to me while there's no support for the iOS side. If we ever change the simulators, we can tweak this value manually.

For consideration maybe we can use the `pixelRatio` name that the [Android driver uses](https://github.com/appium/appium-uiautomator2-driver/blob/35e55d5ccd72a871c830cacc8b62f00bdd9d4a76/lib/driver.js#L212).

Reply:
Addressed in https://github.com/WordPress/gutenberg/pull/55322/commits/5d684e20780e3666aca0b66b763802160a5e6582.